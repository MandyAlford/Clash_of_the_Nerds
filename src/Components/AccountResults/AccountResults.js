import React from 'react';
import './AccountResults.scss';
import PropTypes from 'prop-types';

export const AccountResults = (props) => {
  return (
    <div className='accounts-card'>
      <h2>{props.result.question}</h2>
      <h4>Your Answer: {props.result.userGuess}</h4>
      <h4>Correct Answer: {props.result.correct_answer}</h4>
    </div>
  )
}

AccountResults.propTypes = {
  result: PropTypes.object
}

export default AccountResults
