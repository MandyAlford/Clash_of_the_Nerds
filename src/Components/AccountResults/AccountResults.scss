.accounts-card {
  width: 70%;
  background-color: rgba(255, 255, 255, 0.9);
  border: 5px solid #662324;
  padding: 10px;
  margin: 10px;
  margin-left: 15%;
}
