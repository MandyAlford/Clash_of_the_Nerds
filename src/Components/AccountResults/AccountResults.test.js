import React from 'react';
import { AccountResults } from './AccountResults';
import { shallow } from 'enzyme';

describe('AccountResults', () => {
  it('should match snapshot', () => {
    let wrapper = shallow(<AccountResults
      result={{
        category: 'Mythology',
        type: 'multiple',
        difficulty: 'medium',
        question: 'This Greek mythological figure is the god/goddess of battle strategy (among other things).',
        correct_answer: 'Athena',
        incorrect_answers: ['Ares','Artemis','Apollo'],
        userGuess: 'Athena'
      }}
      />)
    expect(wrapper).toMatchSnapshot()
  })
})
