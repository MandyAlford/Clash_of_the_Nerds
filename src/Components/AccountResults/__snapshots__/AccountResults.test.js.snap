// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountResults should match snapshot 1`] = `
<div
  className="accounts-card"
>
  <h2>
    This Greek mythological figure is the god/goddess of battle strategy (among other things).
  </h2>
  <h4>
    Your Answer: 
    Athena
  </h4>
  <h4>
    Correct Answer: 
    Athena
  </h4>
</div>
`;
